controllers/user.js

```diff
@@ -234,8 +234,29 @@ exports.userListPage = function (aReq, aRes, aNext) {
       pageMetadata(options, ['Flagged Users', 'Moderation']);
     }
   }
-  function render() { aRes.render('pages/userListPage', options); }
-  function asyncComplete(err) { if (err) { return aNext(); } else { preRender(); render(); } }
+  function render() {
+    aRes.render('pages/userListPage', options);
+  }
+  function asyncComplete(aErr) {
+    if (aErr) {
+      aNext();
+      return;
+    }
```

Comment:
MITIGATION: Test why this error fall through is the only one... I am guessing if we have an error in the changed views we're going to 500 and not fail gracefully.